index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Project1</title>
+  </head>
+  <body class="bg-secondary">
+    <div class="container-fluid">
+      <!-- logo & hamburger menu -->
+      <div class="logo-menubar display-flex1">
+        <img src="./images/logo.svg" alt="logo" />
+        <img
+          class="menu-icon"
+          src="./images/menu.svg"
+          alt=""
+        />
+      </div>
+      <!-- end of logo & hamburger menu -->
+      <!-- main body start -->
+      <div class="main display-flex2">
+        <div class="intro">
+          <h1 class="h1">
+            I’m Balqees Ibrahim Glad to see you!
+          </h1>
+          <p>
+            I’m a software developer! I can help
+            you build a product feature or
+            website. Look through some of my work
+            and experience! If you like what you
+            see and have a project you need coded,
+            don’t hesitate to contact me.
+          </p>
+        </div>
+        <a class="connect" href="#"
+          >let's connect</a
+        >
+        <div class="social-icons">
+          <a href="https://twitter.com/qeescode">
+            <img
+              src="./images/twitter.svg"
+              alt=""
+            />
+          </a>
+          <a
+            href="https://www.linkedin.com/in/balqees-ibrahim-911a81204/"
+            class="p1"
+          >
+            <img
+              src="./images/linkedin.svg"
+              alt=""
+            />
+          </a>
+          <a href="#" class="p1">
+            <img
+              src="./images/medium.svg"
+              alt=""
+            />
+          </a>
+          <a
+            href="https://github.com/balqees0325"
+            class="p1"
+          >
+            <img
+              src="./images/github.svg"
+              alt=""
+            />
+          </a>
+          <a href="#" class="p1">
+            <img src="./images/hand.svg" alt="" />
+          </a>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] You should HTML semantic tags instead of using `div`. You can refresh your knowledge in [curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). Also you can check this [article ](https://www.pluralsight.com/guides/semantic-html) why semantic tags are important.

Reply:
Done

Reply:
All corrections effected, What next please?